src/workerd/api/kv.c++

```diff
@@ -424,23 +473,34 @@ jsg::Promise<jsg::JsRef<jsg::JsValue>> KvNamespace::list(
     jsg::Lock& js, jsg::Optional<ListOptions> options) {
   return js.evalNow([&] {
     auto& context = IoContext::current();
+    auto traceSpan = context.makeTraceSpan("kv.list"_kjc);
+    auto userSpan = context.makeUserTraceSpan("kv.list"_kjc);
+    TraceContext traceContext(kj::mv(traceSpan), kj::mv(userSpan));
+
+    traceContext.userSpan.setTag("db.system"_kjc, kj::str("cloudflare.kv"_kjc));
```

Comment:
I don't think this should be set within list itself – see the existing getHttpClient(), where we set this attribute for a number of operations at once.
The value of the tag is not matching what we have there (cloudflare.kv vs. cloudflare-kv, for the operation name we have list vs. kv_list)

Reply:
The naming scheme should always be consistent, everything else is likely to cause confusion, especially when renaming things gets forgotten until code is used in production. I'm not opposed to changing the naming scheme per se, but this should be done for all KV operations, in a commit or PR either before or after adding new tags. Everything else is difficult to review.
We also need to be mindful of keeping a consistent source of truth for this – both the old source of truth and the new document you added use db.system:cloudflare-kv, the tag implementation needs to be consistent with its specification.